### PR TITLE
Handle redirect to vendor system in handleloggedout endpoint

### PR DESCRIPTION
**Pull request by mgunnerud**
## Description
- Added `handleloggedout` endpoint for redirecting to vendor system after user has approved/rejected system user and is logged out.

## Related Issue(s)
- #748 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Authentication/Controllers/LogoutController.cs [hunk 1]
```diff
@@ -84,6 +91,31 @@ public ActionResult Logout()
...
+            CookieOptions opt = new CookieOptions() { Domain = _generalSettings.HostName, Secure = true, HttpOnly = true };
+            Response.Cookies.Delete(_generalSettings.AltinnLogoutInfoCookieName, opt);
+
+            Dictionary<string, string> cookieValues = logoutInfoCookie?.Split('?')
+                .Select(x => x.Split('='))
+                .ToDictionary(x => x[0], x => x[1]);
+
+            if (cookieValues != null && cookieValues.TryGetValue("SystemuserRequestId", out string requestId) && Guid.TryParse(requestId, out Guid requestGuid))
+            {
+                Result<string> redirectUrl = await _requestSystemUser.GetRedirectByRequestId(requestGuid);
```
Comment (TheTechArch) [inline src/Authentication/Controllers/LogoutController.cs:112]: I begge tilfellene her må vi slette cookie så vi ikke ender opp i en loop.

File: test/Altinn.Platform.Authentication.Tests/Controllers/LogoutControllerTests.cs [hunk 2]
```diff
@@ -265,6 +266,49 @@ public async Task Logout_LogedIn_RedirectToIss()
...
+            // Act
+            HttpResponseMessage response = await client.SendAsync(requestMessage);
+
+            // Assert
+            Assert.Equal(System.Net.HttpStatusCode.Found, response.StatusCode);
+
+            IEnumerable<string> values;
+            if (response.Headers.TryGetValues("location", out values))
+            {
+                Assert.Equal("https://smartcloudaltinn.azurewebsites.net/", values.First());
```
Comment (TheTechArch) [inline test/Altinn.Platform.Authentication.Tests/Controllers/LogoutControllerTests.cs:287]: Sjekk at cookie er slettet også

Comment (mgunnerud) [reply test/Altinn.Platform.Authentication.Tests/Controllers/LogoutControllerTests.cs]: Lagt på sjekk for cookie i begge testene

File: src/Authentication/Controllers/LogoutController.cs (hunk 1 above)
Comment (mgunnerud) [reply src/Authentication/Controllers/LogoutController.cs:112]: Cookien slettes i starten av metoden

**Review by TheTechArch — APPROVED** [review]
LGTM

**Pull request merged**